src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql

```diff
@@ -0,0 +1,14 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:moto_masking_account_update_permission
+INSERT INTO permissions(id, name, description) VALUES (49, 'moto-mask-input:update', 'Update Moto Masking Account');
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'super-admin'), (SELECT id FROM permissions WHERE name = 'moto-mask-input:update'), NOW(), NOW());
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'admin'), (SELECT id FROM permissions WHERE name = 'moto-mask-input:update'), NOW(), NOW());
+
+--changeset uk.gov.pay:moto_masking_account_read_permission
+INSERT INTO  permissions(id, name, description) VALUES (50, 'moto-mask-input:read', 'View Moto Masking Account');
```

Comment:
The description should be something like “View MOTO mask input settings”.

Reply:
ok will make that change